### PR TITLE
FEATURE: media content modules

### PR DESCRIPTION
**Pull request by coderimus**
### Description (*)
This pull request provides `MediaContentApi` and `MediaContent` modules described at #964 

### Fixed Issues (if relevant)
#964 

### Manual testing scenarios (*)
N/A

### CODE REVIEW

File: MediaContent/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Magento_MediaContent module
+
+The Magento_MediaContent module module provides implementations for managing relations between content and media files used in that content.
```
Comment (sivaschenko) [inline MediaContent/README.md]:
```suggestion
The Magento_MediaContent module provides implementations for managing relations between content and media files used in that content.
```

File: MediaContent/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Extensibility
+
+Extension developers can interact with the Magento_Magento_MediaContent module. For more information about the Magento extension mechanism, see [Magento plug-ins](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html).
```
Comment (sivaschenko) [inline MediaContent/README.md]:
```suggestion
Extension developers can interact with the Magento_MediaContent module. For more information about the Magento extension mechanism, see [Magento plug-ins](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html).
```

File: MediaContentApi/Api/AssignAssetInterface.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaContentApi\Api;
+
+/**
+ *
```
Comment (sivaschenko) [inline MediaContentApi/Api/AssignAssetInterface.php]: Please add a description to the interfaces and interface methods

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @coderimus , thanks for the PR, please see my review comments

**Comment by coderimus** [comment]
@sivaschenko thank you for your review. I implemented requested changes and also added logic to the newly provided methods.

I have a question: should we return the list of media asset objects from the `\Magento\MediaContent\Model\GetAssetsUsedInContent` or just return the list of raw data from the data storage is applicable for now?

**Comment by sivaschenko** [comment]
@coderimus thanks for updates! I think it's better to return the array of media asset ids from GetAssetsUsedInContent

**Pull request merged**